packages/git/src/node/dugite-git.ts

```diff
@@ -372,6 +399,21 @@ export class DugiteGit implements Git {
         }
     }
 
+    private mapRange(toMap: Git.Options.Range | undefined): string {
+        let range = 'HEAD';
```

Comment:
If `fromRevision` is undefined we should compare against the local state, not HEAD.